content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -1142,6 +1159,35 @@ export class ASRouterAdminInner extends React.PureComponent {
             {this.renderAttributionParamers()}
           </React.Fragment>
         );
+      case "groups":
+        return (
+          <React.Fragment>
+            <h2>Message Groups</h2>
+            <table>
+              <thead>
+                <tr className="message-item">
+                  <td>Enabled</td>
+                  <td>Impressions count</td>
+                  <td>Custom frequency</td>
+                </tr>
+              </thead>
+              {this.state.groups &&
+                this.state.groups.map(({ id, enabled, frequency }, index) => (
+                  <Row key={id}>
+                    <td>
+                      <TogglePrefCheckbox
```

Comment:
The checkbox is meant to show if the groups is enabled and disabled. Trying to toggle it off/on will not change the `disabled` state of the group.
I've considered adding `disabled` attribute to the checkbox but it is also used as a way to clear impressions right now.
Perhaps it can be improved with additional UI (follow up).